source/_components/notify.tibber.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: "Tibber"
+description: "Instructions on how to integrate Tibber within Home Assistant."
+date: 2015-10-04 16:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tibber.png
+ha_category: Notifications
+ha_release: 0.81
```

Comment:
"0.80" Right?

Reply:
Yes.
 I thought the component would not be merged before 0.81